pkg/cmd/run.go

```diff
@@ -136,6 +138,26 @@ doppler run --mount secrets.json -- cat secrets.json`,
 		// retrieve secrets
 		dopplerSecrets := fetchSecrets(localConfig, enableCache, fallbackOpts, metadataPath, nameTransformer, dynamicSecretsTTL)
 
+
+		if cmd.Flag("include").Changed && !cmd.Flag("mount").Changed {
+			var err error
+			noExitOnMissingIncludedSecrets := cmd.Flag("no-exit-on-missing-included-secrets").Changed
+
+			if len(secretsToInclude) == 0 {
+				utils.HandleError(fmt.Errorf("you cannot --include with no secrets"))
+			}
+
+			dopplerSecrets, err = utils.SelectSecrets(dopplerSecrets, secretsToInclude)
```

Comment:
I believe we're just performing a basic filter on the map's keys. I would expect this logic to be more generic rather than having a "generic" utility function with logic that's super specific to this use case. Can you use a generic "filter" function here? That would also be more appropriate to live in our utils than a `SelectSecrets` function. If you do want to encapsulate the entirety of this, including the surrounding logic, you should make a new controller function.